src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/CommonStepDef.scala

```diff
@@ -37,6 +37,10 @@ class CommonStepDef extends BaseStepDef {
     CommonPage.clickOnRadioButton(radioButtonBoolean)
   }
 
+  When("""^the user selects the (.*) radio button: (.*)$""") { (radioButtonBoolean: String, buttonName: String) =>
```

Comment:
Just noticed this as well, Boolean means True or False, so this shouldn't really be labeled as a Boolean.